client/src/pages/ResourceDetail.js

```diff
@@ -8,15 +8,21 @@ import ReactMapboxGl, { Layer, Feature } from 'react-mapbox-gl';
 import { deleteResource, getResourceByID } from '../utils/api';
 import ResourcesBreadcrumb from '../components/ResourcesBreadcrumb';
 
-const days = [
-  'Sunday',
-  'Monday',
-  'Tuesday',
-  'Wednesday',
-  'Thursday',
-  'Friday',
-  'Saturday',
-];
+async function addressToLatLong(address) {
+  const apiLatLong =
+    `${'http://www.mapquestapi.com/geocoding/v1/address?' +
+      'key=QhpXMYz3yy5F0Yg5qZSqGmA2XFMIMRAi&maxResults=5&' +
+      'outFormat=json&location='}${address}}&boundingBox=` +
+    `40.121581,-88.253981,40.098315,-88.205082`;
```

Comment:
We should probably move this to an environment variable, but this can be done later